ext/RMagick/rmutil.c

```diff
@@ -736,6 +736,24 @@ rm_init_magickpixel(const Image *image, MagickPixel *pp)
     GetMagickPixelPacket(image, pp);
 }
 
+/**
+ * Initializes the MagickPixel structure to the specified color.
+ *
+ * No Ruby usage (internal function)
+ *
+ * @param pp the MagickPixel
+ * @param color the color
+ */
+void
+rm_set_magickpixel(MagickPixel *pp, const char *color)
+{
+    ExceptionInfo *exception;
+
+    exception = AcquireExceptionInfo();
+    (void) QueryMagickColor(color, pp, exception);
+    (void) DestroyExceptionInfo(exception);
```

Comment:
You should check the exception raised before destroy.

Reply:
These calls are only done from places where we control the value of `color` and it will not be used for external input. This means that it is very unlikely that an exception will be thrown here. If we would throw an exception here we could potential leak memory in the method that calls this. We could change this into a method that returns a boolean so that caller knows that this failed but I would prefer to ignore the exception (that will prob never be raised).

Reply:
If it doesn't need to check for exceptions in here, it may be better to describe the reason in the comments. Without it, it will bring confuse later.

Reply:
Added a comment.